packages/ui/components/loading-text.tsx

```diff
@@ -3,7 +3,7 @@ import { cn } from '../lib/utils';
 
 export const LoadingText = ({ className }: { className?: string }) => {
   return (
-    <div className={cn('lds-ellipsis', className)}>
+    <div className={cn('lds-ellipsis text-session-text -my-1.5 font-normal', className)}>
```

Comment:
Lol negative margin what could go wrong? 😂

Reply:
haha just a hack to get rid of wasted space on the child element, as it needs to be a font size